src/components/storefront/Navbar/Navbar.js

```diff
@@ -20,44 +38,68 @@ const SubHeading = () => {
 
   return (
     pathname === '/store/cart' && (
-      <Box bg="black" px="105px">
-        <Heading color="white" py={6}>
-          MY SHOPPING BAG
-        </Heading>
+      <Box bg="black">
+        <PageContainer maxW="container.xl" bg="black" py={{ base: 0, lg: 4 }}>
+          <Heading color="white" fontSize={['xl', '2xl', '3xl', '4xl', '5xl']}>
+            MY SHOPPING BAG
```

Comment:
nit: `textTransform="uppercase"` for consistency maybe?

Reply:
dd4daf13eb